rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -1163,4 +1163,34 @@ extern "C" RocalTensor ROCAL_API_CALL rocalToDecibels(RocalContext p_context,
                                                       float reference_magnitude,
                                                       RocalTensorOutputType rocal_tensor_output_type);
 
+extern "C" RocalTensor ROCAL_API_CALL rocalResample(RocalContext p_context,
+                                                    RocalTensor p_input,
+                                                    RocalTensor p_input_resample_rate,
+                                                    RocalTensorOutputType rocal_tensor_output_type,
+                                                    bool is_output,
```

Comment:
Please make rocal_tensor_output_type as the last argument in all the functions, and name it as output_dtype please